components/data_stores/actions/add-update-multiple-records/add-update-multiple-records.mjs

```diff
@@ -22,16 +22,73 @@ export default {
       description: "Enter data you'd like to add as an object containing key-value pairs, or reference existing keys and update the values for those existing records. Refer to your existing keys [here](https://pipedream.com/data-stores/).",
```

Comment:
this description should get updated to
```
Enter data you'd like to add as key-value pairs, or reference an object from a previous step using a custom expression (e.g., `{{steps.data.$return_value}}`).
```

Reply:
Sorry I accidentally changed the wrong description